client/src/components/Faq/FaqContainer.js

```diff
@@ -8,9 +8,24 @@ export const FaqContainer = styled.div`
 `;
 
 export const QuestionsContainer = styled.div`
-  max-width: 80rem;
+  max-width: 70rem;
 `;
 
 export const AnswerContainer = styled.div`
-  margin: 1rem 0;
+  summary::-webkit-details-marker {
+    display: none;
+  }
+  summary {
+  font-size: 1.17em;
```

Comment:
indentation? wonder if prettier / eslint is working automatically

Reply:
don't forget to put in your `client/.env` 
```
EXTEND_ESLINT=true
```

Reply:
I think it might be that my defaults on VSCode are messing with the formatting! This is just another PR for the same issue so I will merge this and update this on the next changes.